tests/foreman/cli/test_contentviewfilter.py

```diff
@@ -145,7 +145,7 @@ def test_create_cvf_description(self):
         self.assertEqual(cvf['description'], description)
 
     @tier1
-    def test_create_cvf_by_cv_name(self):
+    def test_positive_create_by_cv_name(self):
```

Comment:
If you like this naming convention then please update other test names accordingly. thank you.